draft-ietf-teas-yang-path-computation.txt

```diff
@@ -0,0 +1,4704 @@
+
+
+
+
+TEAS Working Group                                          I. Busi, Ed.
+Internet-Draft                                       Huawei Technologies
+Intended status: Standards Track                         S. Belotti, Ed.
+Expires: 28 July 2022                                              Nokia
+                                                     O. Gonzalez de Dios
+                                                              Telefonica
+                                                               A. Sharma
+                                                                  Google
+                                                                  Y. Shi
+                                                            China Unicom
+                                                           D. Ceccarelli
+                                                                Ericsson
+                                                         24 January 2022
+
+
+           A YANG Data Model for requesting path computation
+                draft-ietf-teas-yang-path-computation-17
+
+Abstract
+
+   There are scenarios, typically in a hierarchical Software-Defined
+   Networking (SDN) context, where the topology information provided by
+   a Traffic Engineering (TE) network provider may be insufficient for
+   its client to perform end-to-end path computation.  In these cases
+   the client would need to request the provider to calculate some
+   (partial) feasible paths.
+
+   This document defines a YANG data model for a Remote Procedure Call
+   (RPC) to request path computation.  This model complements the
+   solution, defined in RFC YYYY, to configure a TE tunnel path in
+   "compute-only" mode.
+
+   [RFC EDITOR NOTE: Please replace RFC YYYY with the RFC number of
+   draft-ietf-teas-yang-te once it has been published.
+
+   Moreover this document describes some use cases where a path
+   computation request, via YANG-based protocols (e.g., NETCONF or
+   RESTCONF), can be needed.
```

Comment:
```suggestion
   Moreover, this document describes some use cases where the path
   computation RPCs defined in this document can be used utilizing the
   NETCONF or RESTCONF protocol
```

Reply:
Fixed in https://github.com/rvilalta/ietf-te-path-computation/pull/96/commits/fc6c5456c7cfaf1ec3ffa97477f75847d2fd428f

Co-Authored-By: sergio belotti <sergio.belotti@nokia.com>